test/metabase/upload_test.clj

```diff
@@ -255,8 +256,8 @@
            [int-type         datetime-type    vchar-type]
            [int-type         vchar-type       vchar-type]
            [int-type         text-type        text-type]
-           [int-type         bool-or-int-type int-type]
-           [bool-or-int-type bool-or-int-type bool-or-int-type]
+           [int-type         bool-int-type    int-type]
+           [bool-int-type    bool-int-type    bool-int-type]
```

Comment:
Didn't think past find-and-replace. Will fill in some gaps.

Reply:
Taking a look at the test, it's not clear how the values were chosen, it's far from exhaustive. Since we have tests for the hierarchy structure and the first-common-ancestor function in `ordered-hierarchy-test`, I'm inclined to just delete this test... 

I'll leave off deleting it for now, but I can't think of any interesting `::*float-or-int*` cases that aren't covered elsewhere. Adding that type also means realigning the whole table, I'm out 😆

Reply:
true, makes sense to leave it for now